nestkernel/simulation_manager.cpp

```diff
@@ -813,27 +842,6 @@ nest::SimulationManager::update_()
 }
```

Comment:
Now I am slightly confused about the links in your previous posting because they just open the complete diff.
But you already got the right code snippet in your posting before, concerning the location **outside** of the do-while-loop. And in line 600-607 (in master) the same code exists **within** the do-while-loop.
Concerning the outside position of this code snippet, this would be called in a kind of irregular way in subsequent calls to `run()` (e.g., one run could be for just a fraction of a min-delay interval, and the next for 100 min-delay intervals, and so on...). I just wonder if this may hurt the structural plasticity algorithm. Most likely not, but given the complexity of interacting algorithms in the kernel I am on the paranoid side.